packages/react/src/components/ProgressIndicator/ProgressIndicator-story.js

```diff
@@ -36,7 +47,9 @@ storiesOf('ProgressIndicator', module)
             <Tooltip
               direction="bottom"
               showIcon={false}
-              triggerClassName={`${prefix}--progress-label`}
+              triggerClassName={`${prefix}--progress-label ${spaceEquallyKnob() &&
+                !verticalKnob() &&
+                `${prefix}--progress-label--no-max-width`}`}
```

Comment:
Hmm, this is looking a bit gnarly, and it seems like every end-user would have to manually add this to any tooltip they render inside a tooltip to get `spaceEqually` to work properly, right?

I'm wondering if we can just modify the styles with a single, global style instead of a couple of different classes.